test/stringify.js

```diff
@@ -9,8 +9,8 @@ var hasSymbols = require('has-symbols');
 var emptyTestCases = require('./empty-keys-cases').emptyTestCases;
 var hasBigInt = typeof BigInt === 'function';
 
-test('stringify()', function (t) {
-    t.test('stringifies a querystring object', function (st) {
+test('stringify()', function(t) {
+    t.test('stringifies a querystring object', function(st) {
```

Comment:
not sure why all these spaces are being removed; make sure all autoformatting in your editor is disabled except for eslint.